float32  speed
float32  radius
float32  angle
geometry_msgs/PoseStamped   robot_pose



